### DISCUSSION TITLE
LDAP support

### DISCUSSION THREAD

**Discussion by paulo-roger** [JulianPrieber/littlelink-custom#815] [Feature requests]
I think it would be nice to have ldap support so all my users of other services could automatically have an account in my linkstack instance.

**Comment by JulianPrieber** [comment]
We currently do not support LDAP in any way.
We do not have plans to add, this, however we may add an API in a future revision that could accomplish what you're looking for.

**Reply by paulo-roger** [reply]
interesting. thanks!
